### PR TITLE
Modify duplicate err msg

### PR DESCRIPTION
**Pull request by zhengguoyong**
modify duplicate err msg.

### CODE REVIEW

**Comment by ashcrow** [comment]
:+1: this clarifies between the two error cases.

File: pkg/kubelet/dockertools/manager.go [hunk 1]
```diff
@@ -1873,7 +1873,7 @@ func (dm *DockerManager) SyncPod(pod *api.Pod, runningPod kubecontainer.Pod, pod
 		// Call the networking plugin
 		err = dm.networkPlugin.SetUpPod(pod.Namespace, pod.Name, podInfraContainerID)
 		if err != nil {
-			glog.Errorf("Failed to create pod infra container: %v; Skipping pod %q", err, podFullName)
+			glog.Errorf("Failed to setup pod networkplugin: %v; Skipping pod %q", err, podFullName)
```
Comment (vishh) [inline pkg/kubelet/dockertools/manager.go]: nit: `Failed to setup networking for pod %q using network plugins: %v; Skipping pod`

**Comment by ashcrow** [comment]
Shippable failure is:

```
FAIL	k8s.io/kubernetes/contrib/mesos/pkg/offers	300.185s
```

and only failed in 1 of the 2 runs. Seems like a flake failure.

**Comment by zhengguoyong** [comment]
tnanks for your review, 
i changed it , please review again,
thanks.
@vishh
@ashcrow

**Comment by ashcrow** [comment]
This looks good and @vishh's nit is fixed.

**Comment by zhengguoyong** [comment]
please review, thanks.

**Comment by ashcrow** [comment]
@vishh bump

**Comment by vishh** [comment]
LGTM. Thanks for the fix @zhengguoyong

**Comment by ashcrow** [comment]
e2e flake

```
[Fail] hostPath [It] should give a volume the correct mode [Conformance] 
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/util.go:1118
```

**Comment by vishh** [comment]
ok to test

**Pull request merged**